packages/flutter/lib/src/widgets/shortcuts.dart

```diff
@@ -386,6 +386,44 @@ class ShortcutManager extends ChangeNotifier with Diagnosticable {
 /// when invoking an [Action] via a keyboard key combination that maps to an
 /// [Intent].
 ///
+/// {@tool dartpad --template=stateful_widget_material}
+/// ```dart imports
```

Comment:
Right before this line, I think we should add some text to describe how to use this sample:

```dart
/// Here, we will use a [Shortcuts] and [Actions] widget to add and remove from a counter. 
/// This can be done by creating a child widget that is focused and pressing the logical key 
/// sets that have been defined in [Shortcuts] and defining the actions that each key set 
/// performs.
```